app/views/govuk_publishing_components/components/docs/taxonomy_navigation.yml

```diff
@@ -0,0 +1,102 @@
+name: "Taxonomy navigation"
+description: "Sidebar navigation for displaying on pages tagged to the GOV.UK taxonomy."
+body: |
+  Accepts an array of items. These items are typically the list of taxons the current
```

Comment:
Worth noting that this component is never used directly, only via the contextual sidebar component.

Reply:
And maybe that it should be merged with the related_navigation component eventually.